include/electionguard/discrete_log.hpp

```diff
@@ -68,15 +68,32 @@ namespace electionguard
         }
 
         /// <Summary>
-        /// Get the discrete log value for the given element
+        /// Get the discrete log value for the given element.
+        /// This override assumes that the generator G() is being used as the base for exponentiations.
+        ///
+        /// In Electionguard 2.0 this method is deprecated in favor of the override that
+        /// allows for the caller to specify the base for exponentiations.
         /// </Summary>
         static uint64_t getAsync(const ElementModP &element);
```

Comment:
this is actually deprecated but i couldn't find a consistent cross platform way to mark it as such without creating a bunch of work for myself. maybe you guys know of something off the top of your head

Reply:
It's kind of messy but also not sure it adds much value. `__declspec(deprecated)` macro could do it

Reply:
__declspec is usually Windows only and not cross platform